### ISSUE TITLE
Failed to start machine (Windows)

### ISSUE THREAD

**Issue by jakirkham** [boot2docker/boot2docker#999]
Using boot2docker 1.7.1 (latest), Git 1.9.5, and Windows 7, I am unable to start the machine using `boot2docker up`. Full log here ( https://gist.github.com/jakirkham/4144f4a3fd4cd7ec1d14 )

**Comment by jaredpetersen** [comment]
Boot2Docker Owners/Maintainers:

I have experienced the same problem (and unknowingly created a duplicate issue). Please check out #1010 if you would like more information.

**Comment by jakirkham** [comment]
So, I completely wiped the installation. Removed all copies of Git, MinGW, boot2docker, and VirtualBox with the uninstaller and deleted all remaining files manually. Also, I removed the `.boot2docker` directory in my home directory and all of its contents. Did a fresh installation and ran "Boot2Docker Start" and was able to get docker up and running again. However, I now get a new issue ( #1017 ).

**Comment by jaredpetersen** [comment]
Interesting, I wiped everything (minus Git) as well as part of my troubleshooting, but wasn't able to achieve the same results.

**Comment by jakirkham** [comment]
I should add that I downloaded a new copy of the installer. Maybe it is worth wiping Git, as well.

**Comment by jaredpetersen** [comment]
I've downloaded many, many copies of the installer without luck but I may have to try removing Git as well. That seems like a really weird thing to have to do though.

**Comment by jakirkham** [comment]
Weird, but maybe needed. Also, make sure you remove the `.boot2docker` directory from your user directory.

**Comment by jaredpetersen** [comment]
Uninstalled, removed ``.boot2docker`` directory, and reinstalled. No luck. I'm not going to uninstall git to get this to work because it'll take too long to get the access to reinstall it on my work laptop.

**Comment by jakirkham** [comment]
Any luck?

**Comment by jaredpetersen** [comment]
No, I haven't tried anything since. I ended up having to move on to something else entirely, so I never went back to deal with this. Short of uninstalling Git, I'm out of options anyways.

**Comment by AkihiroSuda** [comment]
I think Boot2Docker CLI is no longer maintained.
I suggest closing this issue.

People who are hitting this issue can use Docker Machine instead: https://docs.docker.com/windows/step_one/
